raptorWeb/staffapps/views.py

```diff
@@ -0,0 +1,75 @@
+from django.shortcuts import render
```

Comment:
Dupe from `staffapps.models` 

General question/suggestion for a possibly better system. In what ways could we only use a single View `AppView` for all forms, and still allow different questions and different types of forms that still behave the same way for the end user?

Reply:
This is something for me to look at later